extensions/ql-vscode/src/queries-panel/query-node.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * A node in the tree of queries. This will be either a `QueryDirectory` or a `QueryFile`.
+ */
+export abstract class QueryNode {
```

Comment:
What do you think about leaving these classes out for now and not including the query node as part of the tree view item? I'm not convinced we'll need it.

Reply:
Yes, good point. Removed that in https://github.com/github/vscode-codeql/pull/2418/commits/61d4c9c695f603d43ef0ab73871b0496fbb8a1e4 💥